packages/component-submission/client/pages/SubmissionWizard.js

```diff
@@ -102,6 +103,7 @@ export const SubmissionWizard = ({
               <Box my={5}>
                 <ProgressBar currentStep={currentStep} />
               </Box>
+              <FormH2>{STEP_NAMES[currentStep]}</FormH2>
```

Comment:
Does this need applying to all other steps (not including files) too?

Reply:
this will apply to all steps, its in the top level of the form